Automated testing (#32)
l7mp/learning-go: README.md

### Test

#### Manual testing
At any point in time you can test your solutions as follows.

``` console
make test
```

#### Automated testing remotely

> You have the option to test your homeworks using GitHub Actions.

Reviewer: ```suggestion
You have the option to test your **homeworks** using GitHub Actions.
```

To do so, you just have to commit your local changes and push them to your master branch on your forked repository.

```console
git add .
git commit -m '<your commit message>'
git push
```
This will start an automatic testing pipeline that you can check in the GitHub GUI.

> You can also check your labs exercies, to do so, navigate to the GitHub repository (your repo), click `Actions`, click `Run workflow`, pass in the parameters and click `Run workflow`.

Reviewer: ```suggestion
You can also check your **labs**, to do so, navigate to your GitHub repository, click `Actions`, click `Run workflow`, set parameters and click `Run workflow`.
```
